dependencies.props

```diff
@@ -17,18 +17,14 @@
     <CoreClrCurrentRef>96793f61d3d482bcb59326c18a75bce39971c55f</CoreClrCurrentRef>
   </PropertyGroup>
 
-  <!-- Auto-upgraded properties for each build info dependency. -->
+  <!-- Tests/infrastructure dependency versions. -->
   <PropertyGroup>
     <CoreFxExpectedPrerelease>servicing-25727-01</CoreFxExpectedPrerelease>
```

Comment:
slightly orthogonal but we should probably eliminate this and instead include the exact package versions.

Reply:
Agreed, I'll add this change. Makes sense to me to do it in this PR since it'll fall in orchestrator scope once it builds tests.